src/ui/profile/mod.rs

```diff
@@ -15,6 +15,7 @@ use crate::{
 use super::TimelineView;
 
 pub struct ProfileView<'a> {
+    pubkey: Pubkey,
```

Comment:
let's not copy 32 bytes of owned data every frame, let's convert these to references